### PR TITLE
Add doc for /me endpoint

### PR DESCRIPTION
**Pull request by hklee93**

### CODE REVIEW

File: platform_API.md [hunk 1]
```diff
@@ -50,6 +50,19 @@ Authorization: Key <your Server API Key>
 
 ## API Reference
 
+### Authentication
+
+#### Access a user's resource
```
Comment (CMEONE) [inline platform_API.md:55]: Maybe add a colon here? "Access a user's resource:"

File: platform_API.md [hunk 2]
```diff
@@ -50,6 +50,19 @@ Authorization: Key <your Server API Key>
 
 ## API Reference
 
+### Authentication
+
```
Comment (CMEONE) [inline platform_API.md:54]: Might be good to include a "Base path: `/me`" similar to the Payments documentation below to improve consistency.

**Review by CMEONE — CHANGES_REQUESTED** [review]

File: platform_API.md (hunk 1 above)
Comment (hklee93) [reply platform_API.md]: Thanks for catching this!

File: platform_API.md (hunk 2 above)
Comment (hklee93) [reply platform_API.md:54]: I could be wrong but I don't think we plan to use `/me` as the base path.

Comment (CMEONE) [reply platform_API.md:54]: True, it seems like there wouldn't be any url parameters so it's probably fine without a base path.

**Review by CMEONE — APPROVED** [review]
LGTM!

**Comment by Son120783** [comment]
Ok

**Comment by lazike66** [comment]
Up date

**Review by Madkunt — APPROVED** [review]

File: platform_API.md (hunk 2 above)
Comment (SarianSow) [reply platform_API.md:54]: Nice
I wish i understand about this

**Comment by SarianSow** [comment]
Nice project

File: platform_API.md (hunk 1 above)
Comment (420aniket) [reply platform_API.md]: Payment in acaunt

**Comment by marjuki022** [comment]
Nice project

**Review by OhArthits — APPROVED** [review]

**Comment by Senaherbalist** [comment]
ee9d5783913ac0d142b6daeea471576b69ad69e9

File: platform_API.md (hunk 1 above)
Comment (SHADYKIMBUM) [reply platform_API.md]: cbttdpfyhsj2guu6rcnzwctfijniocixzovlvjlxrvadppnedrxb1pnymcikqooy

**Review by GOFFERU — APPROVED** [review]
Thanks for suggested to me.

**Comment by haticecan37** [comment]
387117eff8755ba47d7787e3c083bca3029d442c

**Comment by haaami01** [comment]
Thanks for the suggestion too!

**Review by evandrofgarcia — COMMENTED** [review]
Amazing

**Review by evandrofgarcia — COMMENTED** [review]
Essentially opportunity

**Review by Juma-creator — APPROVED** [review]

**Review by Kamicio1234 — APPROVED** [review]

File: SDK_reference.md [hunk 3]
```diff
@@ -34,6 +34,12 @@ to configure this and view your Sandbox URL.
...
 ## Authentication
 
+> **Warning:** The user information obtained with this method should not be passed to your backend and should
+> only be used for presentation logic (e.g displaying the user’s username).
+> **On your backend, use the Platform API as the source of truth.** You can verify the user's
+> identity by requesting the /me endpoint from your backend, using the access token obtained with this method.
+
+
 ```typescript
 Pi.authenticate(scopes: Array<string>, onIncompletePaymentFound: Function<PaymentDTO>): Promise<AuthResult>
```
Comment (Banjosa07) [inline SDK_reference.md:44]: Good

**Review by Banjosa07 — APPROVED** [review]

**Review by Kamicio1234 — APPROVED** [review]

**Review by Kamicio1234 — APPROVED** [review]

**Comment by OakTechn** [comment]
Nice

**Review by OakTechn — APPROVED** [review]

**Review by Kamicio1234 — APPROVED** [review]

File: platform_API.md (hunk 2 above)
Comment (brianstohl87-cpu) [reply platform_API.md:54]: Not sure but I think I'm good now

**Pull request merged**